Libs/Mesh/PreviewMeshQC/FECVDDecimationModifier.cpp

```diff
@@ -127,9 +121,12 @@ bool FECVDDecimationModifier::Initialize(FEMesh* pm)
 		if (pm->Element(i).IsSelected()) selectedFaces.push_back(i);
 	}
 
+	// create random number generator

```

Comment:
Overall, this function seems much cleaner with const variables marked as such. But the comment above regarding using the same rng seed for each call is still an issue.

Reply:
Done